ruby/vampires.rb

```diff
@@ -52,5 +53,6 @@
     puts result
     i += 1
 end
+
```

Comment:
@emgreen33 this is a solid work.  You are doing a great job at protecting the flow of the file through conditionals.  I look forward to your method work this week as we will be teaching how to encapsulate actions to smaller secured methods.  If you have the chance reflect on your method and data structure work this week and see if there are ways to organize this file into something set up through methods calls.  If you don't get to that no worries.

Like I said this is a good submission!